### PR TITLE
OP-609 Settings->Price List->Manage Lists->New/Edit dialog

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-609

Includes some code and formatting improvements.

### CODE REVIEW

**Review by mwithi — CHANGES_REQUESTED** [review]
CURRENCY FIELD shown cropped
image

**Comment by dbmalkovsky** [comment]
Interesting; I don't see the same thing on Linux
image

**Comment by mwithi** [comment]
> And even more interesting is that I didn't change any code related to the dialog in this PR 😄

sorry, I thought it was concerning ListBrowser also, so it is a different case...

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

**Comment by mwithi** [comment]
The PR is approved, if you want to address the ListBrowser problem in this PR I can wait, otherwise OK to merge for me

File: src/main/java/org/isf/priceslist/gui/ListBrowser.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ public void listUpdated(AWTEvent e) {
 			MessageBundle.getMessage("angal.common.description.txt").toUpperCase(),
 			MessageBundle.getMessage("angal.priceslist.currency.col").toUpperCase()
 	};
-	private int[] columnWidth = {100, 100, 200, 100};
+	private int[] columnWidth = {100, 100, 200, 200};
```
Comment (mwithi) [inline src/main/java/org/isf/priceslist/gui/ListBrowser.java:81]:
```suggestion
	private int[] columnWidth = {100, 100, 200, 100};
```

File: src/main/java/org/isf/priceslist/gui/ListBrowser.java [hunk 2]
```diff
@@ -89,26 +89,21 @@ public void listUpdated(AWTEvent e) {
 	public ListBrowser() {
 		myFrame = this;
 		initComponents();
+		pack();
 		setLocationRelativeTo(null);
 	}
 
 	private void initComponents() {
-		add(getJScrollPaneTable(), BorderLayout.CENTER);
+		add(getJScrollPaneTable(), BorderLayout.NORTH);
```
Comment (mwithi) [inline src/main/java/org/isf/priceslist/gui/ListBrowser.java:97]:
```suggestion
		add(getJScrollPaneTable(), BorderLayout.CENTER);
```

**Review by mwithi — APPROVED** [review]
With `pack()` previous values looks better IMHO

File: src/main/java/org/isf/priceslist/gui/ListBrowser.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/priceslist/gui/ListBrowser.java:97]: Done

Comment (mwithi) [reply src/main/java/org/isf/priceslist/gui/ListBrowser.java]: Thanks!

**Pull request merged**